mitosheet/src/pro/taskpanes/ConditionalFormatting/ConditionalFormattingTaskpane.tsx

```diff
@@ -130,6 +130,8 @@ const ConditionalFormattingTaskpane = (props: ConditionalFormattingTaskpaneProps
                 userProfile={props.userProfile} 
                 requiresPro
                 requiresProMessage="Conditional formatting is a Mito Pro feature. Please upgrade to use this feature."
+                mitoAPI={props.mitoAPI}
+                featureName="conditional_formatting"
```

Comment:
Can we collapse these into one type. So the props can be:

```
requiresPro?: {
    message: string,
    featureName: string,
    mitoAPI: MitoAPI
}
```

And same for enterprise. That way you don't have to remember to provide all of these optional arguments together.